numbers.js

```diff
@@ -1 +1,16 @@
 //This file contains Javascript functions that is operated on numbers.
+
+function fibo(range){
+    let f1=0
+    let f2=1
+    let f3;
+    let arr=[0,1]
```

Comment:
The function looks good, but what is the expected output when the range is 1? I think the function would return 0,1 for inputs 0 and 1 which is incorrect. Please make the changes to address this edge cases.

Reply:
Fixed.